src/services/services.ts

```diff
@@ -1807,6 +1807,15 @@ namespace ts {
             return range && createTextSpanFromRange(range);
         }
 
+        function getSpanForPosition(fileName: string, position: number): TextSpan {
+            synchronizeHostData();
+
+            const sourceFile = getValidSourceFile(fileName);
+            const node = getTouchingPropertyName(sourceFile, position, /*includeJsDocCcomment*/ false);

```

Comment:
>false [](start = 96, length = 5)

Do we support GTD in doc comments?  (I don't know, but I believe C# does.)

Reply:
We do support it, fixed.